status_page.html

```diff
@@ -62,16 +87,18 @@ <h2>
             </h2>
             <svg class="status-icon" color: #D11" xmlns="http://www.w3.org/2000/svg" fill="currentColor" color="white" class="bi bi-x-circle-fill" viewBox="0 0 16 16"> <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM5.354 4.646a.5.5 0 1 0-.708.708L7.293 8l-2.647 2.646a.5.5 0 0 0 .708.708L8 8.707l2.646 2.647a.5.5 0 0 0 .708-.708L8.707 8l2.647-2.646a.5.5 0 0 0-.708-.708L8 7.293 5.354 4.646z" fill="red"></path> </svg>
         </div>
-        <div  id="server-up">
+        <div id="server-up">
             <h2>
                 The server is up!
             </h2>
             <svg class="status-icon" style="color: #1D1;" xmlns="http://www.w3.org/2000/svg" fill="currentColor" class="bi bi-check-circle-fill" viewBox="0 0 16 16"> <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z" fill="#23d100"></path> </svg>
-        </div>
-        <h3 id="players">
+            <h3 id="players">
             There are <span id="player-count"></span> players right now!
         </h3>
         <h3 id="no-players">No one is playing right now!</h3>
+        <a href="steam://rungameid/892970//+connect 96.224.208.176:2456">Join Server</a>
```

Comment:
ah, you probably didnt want to have commited this.  Its probably not a huge problem, but I was trying to be careful to not put your home IP in a public repo.  We wanted to change this to either be baked in by the build process or pulled by an api on load, yea?